tests/integration/golang/mlflow/run/log_batch_test.go

```diff
@@ -123,6 +123,14 @@ func (s *LogBatchTestSuite) TestParams_Ok() {
 	})
 	require.Nil(s.T(), err)
 
+	// create preexisting param (other batch) for conflict testing
+	_, err = s.ParamFixtures.CreateParam(context.Background(), &models.Param{
+		RunID: run.ID,
+		Key:   "key1",
+		Value: "value1",
+	})
+	assert.Nil(s.T(), err)
```

Comment:
ah shoot, we have merged #584 in the meantime, so you should use `require.Nil` instead of `assert.Nil` to be consistent!
sorry for the extra work 😞

Reply:
done